docs/stack/use-cases/_index.md

```diff
@@ -0,0 +1,417 @@
+---
```

Comment:
I'm not sure all should be on the same page.
Why not have use-cases under RedisGraph, RediSearch, etc., and here just point to all these pages.

Reply:
Good idea! I think we need intro paragraph for each section and link to the respective modules. But I also see the value in having them all on the same page.

Reply:
Let's revisit in the next quarter. I think it may be good to have them on the same page.